packages/apps/human-app/frontend/src/shared/helpers/get-notification-icon-by-type.tsx

```diff
@@ -0,0 +1,17 @@
+import CheckCircleIcon from '@mui/icons-material/CheckCircle';
+import ErrorIcon from '@mui/icons-material/Error';
+import { type TopNotificationType } from '@/shared/types/notifications';
+import { colorPalette as lightColorPalette } from '@/shared/styles/color-palette';
+
+export const getNotificationIconByType = (
```

Comment:
It's used only (and highly likely will be used only) in `NotificationProvider`, so no need to move it to `helpers`, it can live in the same file